photon-client/src/components/settings/ApriltagControlCard.vue

```diff
@@ -0,0 +1,95 @@
+<script setup lang="ts">
+import CvSlider from "@/components/common/cv-slider.vue";
+import { useCameraSettingsStore } from "@/stores/settings/CameraSettingsStore";
+import { useStateStore } from "@/stores/StateStore";
+import { useSettingsStore } from "@/stores/settings/GeneralSettingsStore";
+import { Euler, Quaternion } from "three";
+
+function quatToEuler(quat) {
```

Comment:
Add type `Quaternion` from `PhotonTrackingTypes` and (nit) convert this to arrow function declaration for convention